README.md

```diff
@@ -14,6 +14,9 @@ exports.config = {
     // Output json and html will go in this folder.
     outdir: 'timelines',
 
+    // Set the name of the html file. Defaults to index.html.
+    htmlFile: 'results.html'
```

Comment:
My problem with the name `html` is that it's not clear if we're talking about an HTML file for the app being tested or the result file